api/src/opentrons/protocol_api/core/engine/labware.py

```diff
@@ -120,7 +120,7 @@ def get_next_tip(
     ) -> Optional[str]:
         return self._engine_client.state.tips.get_next_tip(
             labware_id=self._labware_id,
-            use_column=num_tips != 1,
+            tip_amount=num_tips,
```

Comment:
If we're going to pass this argument through, lets keep it named the same thing so our brains can do one less thing

```suggestion
            num_tips=num_tips,
```